charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -332,6 +332,7 @@ node:
   loggingFormat: text
   logLevel: 2
   priorityClassName:
+  additionalArgs: []
```

Comment:
Question: why expose this parameter, given it introduces the following overhead:

> If i specify a value via additionalArgs and also through its dedicated helm parameter, what takes precedence?

Will chart changes require additional guards for every single parameter, or do we already have a reliable way to determine what the behavior is?

If the only reason for adding this is consistency with controller parameters, or add-on schema, I think removing the invalid add-on exposed parameter should be considered as an alternative to adding this parameter.

Besides the question above, this PR lgtm.

Reply:
I think consistency with all other containers is a valid enough reason. Having this parameter would have prevented the need for this re-release, and it is already advertised in add-on schema (which we cannot remove because that is a customer facing change, even if it currently doesn't do anything).

Reply:
+1 to exposing, there's no reason for this container to be special and not allow specifying `additionalArgs` - it's a weird and inconsistent user experience.